examples/cifar10_deepspeed_te.py

```diff
@@ -0,0 +1,406 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""The deepspeed cifar10 exampe using MS-AMP. It is adapted from official deepspeed example.
+
+The only change is add "from msamp import deepspeed" and use FP8VisionTransformer instead of orignial model Net.
```

Comment:
Please explain the source of model definition and training script.
For example, `Adapted from xxx`

Reply:
Done. Explained the model definition in comment.